config/routes.rb

```diff
@@ -2,14 +2,14 @@
   
   root "photos#index"
   
+  
   devise_for :users
   
   resources :likes
   resources :follow_requests
   resources :comments
   resources :photos
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-
+  
+  get"/:username" => "users#show"
```

Comment:
Hi Dylan, your username route is missing the route alias. Please add that and resubmit. Everything else looks good.